src/Analyzers/CSharp/CodeFixes/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -170,8 +170,8 @@ private static SyntaxTriviaList Expand(List<StatementSyntax> result, UsingStatem
         private static LocalDeclarationStatementSyntax Convert(UsingStatementSyntax usingStatement)
         {
             return LocalDeclarationStatement(
-                usingStatement.AwaitKeyword,
-                usingStatement.UsingKeyword,
+                usingStatement.AwaitKeyword.WithTrailingTrivia(ElasticSpace),
+                usingStatement.UsingKeyword.WithTrailingTrivia(ElasticSpace),
```

Comment:
❗ The code fix application is _required_ to work without this. This is an error in the formatter's handling of elastic trivia.

Reply:
also, this would blow away any actual trailing trivia already tehre.

Reply:
@sharwell i'm not sure if that's true.  the formatter doesn't act as a normalizer/fixer in the cast of syntax *without* any sort of trivia on it.  So if you generate a tree with two tokens taht abut (illegally) and do *not* have any trivia whatsoever, i don't believe the formatter does anything.

@heejaechang could say more for certain.

Reply:
Ah, that's right. If you generate the `await` and/or `using` tokens, it will have an elastic marker for trivia. If you reuse the ones that already exist, you'll need to add the elastic markers. In this case, I'd leave the `await` keyword as-is, and just prepend leading/append trailing the `using` keyword with ElasticMarker.

Reply:
yep, what Cyrus said is right. and as Sam said, generated token already has elastic trivia attached.

Reply:
> In this case, I'd leave the `await` keyword as-is, and just prepend leading/append trailing the `using` keyword with ElasticMarker.

I'll revise this to say we _only_ need to append trailing trivia (i.e. `usingKeyword.WithAppendedTrailingTrivia(ElasticMarker)`). Since the `await` and `using` were already distinct and the code fix doesn't alter their relation, it's impossible for it to need an additional space at this location.

Reply:
yes.  that sounds right to me.

Reply:
```suggestion
                usingStatement.AwaitKeyword,
                usingStatement.UsingKeyword.AppendTrailingTrivia(ElasticMarker),
```

Reply:
Thanks @sharwell, @CyrusNajmabadi, and @heejaechang ! Fixed it now.